src/Components/Ignitor/src/BlazorClient.cs

```diff
@@ -359,7 +359,19 @@ public async Task<bool> ConnectAsync(Uri uri, bool connectAutomatically = true,
             HubConnection.On<string>("JS.Error", OnError);
             HubConnection.Closed += OnClosedAsync;
 
-            await HubConnection.StartAsync(CancellationToken);
+            for (var i = 0; i < 10; i++)
+            {
+                try
+                {
+                    await HubConnection.StartAsync(CancellationToken);
```

Comment:
@BrennanConroy will this leave the hubconnection in an invalid state if the call fails?

Reply:
It will leave it in a disconnected state

Reply:
I'm explictly asking if re-using the same hubconnection for the retries is OK

Reply:
Yes.

Although I'd love to know why you think this is needed. SignalR has 100's of tests that use the HubConnection to connect to a server and doesn't need to use retries.

Reply:
I'm just being cautious here. Based on the error I have not much idea of what can be going wrong. Other than the server is taking longer to start?